src/wasm-lib/kcl/src/std/args.rs

```diff
@@ -181,47 +181,39 @@ impl Args {
         Ok(())
     }
 
-    fn make_user_val_from_json(&self, j: serde_json::Value) -> Result<KclValue, KclError> {
-        Ok(KclValue::UserVal(crate::executor::UserVal {
+    fn make_user_val_from_json(&self, j: serde_json::Value) -> KclValue {
```

Comment:
I realized this function always returned Ok, so I removed the Result wrapper from the return type. That change cascaded through to some other functions.